roles/provision_virtual_esxi/tasks/main.yml

```diff
@@ -11,25 +11,21 @@
     fail_msg: Variable must be set when using this role.
 
 - name: Check For Missing Hosts
-  community.vmware.vmware_guest_info:
+  vmware.vmware.guest_info:
     hostname: "{{ provision_virtual_esxi_hostname }}"
     username: "{{ provision_virtual_esxi_username }}"
     password: "{{ provision_virtual_esxi_password }}"
     port: "{{ provision_virtual_esxi_port | default(omit) }}"
-    datacenter: "{{ provision_virtual_esxi_datacenter | default('') }}"
     validate_certs: "{{ provision_virtual_esxi_validate_certs | default(omit) }}"
     name: "{{ _esxi.name }}"
-  failed_when: false
```

Comment:
IIUC the failure [link](https://main-jenkins-csb-aap.apps.ocp-c1.prod.psi.redhat.com/job/AAPQA/job/Content/job/Cloud/job/VMware_ops/job/Content-VMwareOps-Collection-Tests/21/testReport/tests.content.vmware_ops.test_vmware_ops/TestContentVMwareOps/test_content_integration_vmware_ops_vmware_ops_vcenter_host_connection_test_/), without this line role will fail in case `"{{ _esxi.name }}"` doesn't exist